opa/opa.go

```diff
@@ -93,30 +95,34 @@ func (a *restAuthorizer) Authorize(subject string, groups []string, permission r
 			"status", res.Status,
 		)
 
-		return false
+		return res.StatusCode, false
```

Comment:
I just wonder if we should take specific actions depending on the class of the errors. For example, would a logline also helpful for debugging purposes when we receive a 500 error? This would surface the configuration errors earlier.

Reply:
@kakkoyun I don't quite get your point here. There is a log line above in L91.

Reply:
Sorry I haven't seen that, it was folded 🙈